lib/scheduled_jobs/update_cmc_prices.rb

```diff
@@ -15,15 +15,15 @@ def perform(metadata)
   def perform_prices(id)
     case id
     when 1 # Top 1-100
-      CoinMarketCapPro::UpdateTickerService.call(start: 1, limit: 100)
+      CoinMarketCapPro::UpdateTickerService.call(start: 1, limit: 100, healthcheck_url: ENV.fetch('HEALTHCHECK_SNAP_PRICES1'))
     when 2 # Top 101-200
-      CoinMarketCapPro::UpdateTickerService.call(start: 101, limit: 100)
+      CoinMarketCapPro::UpdateTickerService.call(start: 101, limit: 100, healthcheck_url: ENV.fetch('HEALTHCHECK_SNAP_PRICES2'))
     when 3 # Top 201-500
-      CoinMarketCapPro::UpdateTickerService.call(start: 201, limit: 300)
+      CoinMarketCapPro::UpdateTickerService.call(start: 201, limit: 300, healthcheck_url: ENV.fetch('HEALTHCHECK_SNAP_PRICES3'))
     when 4 # Top 501-1000
-      CoinMarketCapPro::UpdateTickerService.call(start: 501, limit: 500)
+      CoinMarketCapPro::UpdateTickerService.call(start: 501, limit: 500, healthcheck_url: ENV.fetch('HEALTHCHECK_SNAP_PRICES4'))
     when 5 # Top 1001+
-      CoinMarketCapPro::UpdateTickerService.call(start: 1001, limit: 5000) # 5000 is CMC's max limit
+      CoinMarketCapPro::UpdateTickerService.call(start: 1001, limit: 5000, healthcheck_url: ENV.fetch('HEALTHCHECK_SNAP_PRICES5')) # 5000 is CMC's max limit
     end
```

Comment:
Could be worth adding `default` case statement to check if unexpected `id` is passed in and throw some sort of error / print

Reply:
For now, given that the call from prices can only be 1..5, I think it seems unnecessary.